iambic/plugins/v0_1_0/generic_git_provider/aws_lambda_handler.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
+
+import os
+import shutil
+import tempfile
+import traceback
+from functools import cache
+from typing import Callable
+
+import boto3
+import yaml
+from botocore.exceptions import ClientError
+
+import iambic.core.utils
+import iambic.plugins.v0_1_0.github.github
+from iambic.core.logger import log
+from iambic.plugins.v0_1_0.generic_git_provider.generic_git_client import (
+    create_git_client,
+)
+from iambic.plugins.v0_1_0.github.github import (
+    _handle_import,
+    get_session_name,
+    iambic_app,
+)
+
+GENERIC_GIT_PROVIDER_SECRET_ARN = os.environ.get("GENERIC_GIT_PROVIDER_SECRET_ARN")
+
+
+def run_handler(event=None, context=None):
```

Comment:
nit: This is code we'll need to maintain separately from the other Lambda? Is `detect` a v2 thing? Main concern is having to update lambda code in 2 places  when we make any changes, but non-blocking.

Reply:
* detect will be a v2 thing. 
* brutal refactor requires high test coverage. high test coverage takes time. it's open for discussion.